src/tag.rs

```diff
@@ -1358,6 +1360,27 @@ impl<'a> Tag {
                 }
             })
     }
+
+    /// Reads AIFF file and returns ID3 Tag from it
+    pub fn read_from_aiff(path: impl AsRef<Path>) -> crate::Result<Tag> {
+        match aiff::load_aiff_id3(path.as_ref().to_str().unwrap()) {
+            Ok(t) => {
+                match t {
+                    Some(tag) => Ok(tag),
+                    None => Err(Error::new(ErrorKind::NoTag, "Missing ID3 tag!"))
+                }
+            },
+            Err(_) => Err(Error::new(ErrorKind::Parsing, "Failed loading AIFF file!"))
+        }
+    }
+
+    /// Overwrite AIFF file ID3 chunk
+    pub fn write_to_aiff(&self, path: impl AsRef<Path>, version: Version) -> crate::Result<()> {
+        match aiff::overwrite_aiff_id3(path.as_ref().to_str().unwrap(), &self, version) {
+            Ok(_) => Ok(()),
+            Err(_) => Err(Error::new(ErrorKind::Parsing, "Failed writting to AIFF file!"))
```

Comment:
This does not seem like the correct ErrorKind to use here. `ErrorKind::Io` is more appropriate here.

But how about using this crate's Error types in the aiff module?

Reply:
will look at proper error handling later.